root/static/scripts/edit/externalLinks.js

```diff
@@ -1261,7 +1272,7 @@ export function parseRelationships(
         rawUrl: target.name,
         relationship: data.id,
         submitted: true,
-        type: data.linkTypeID ?? null,
+        type: parseInt(data.linkTypeID, 10) ?? null,
```

Comment:
So the server is sending these as strings?  Seems like something we should fix on the server.

Reply:
I found this when I was testing seedings, they're not parsed to int after submitting.

Reply:
I'd guess we should change https://github.com/metabrainz/musicbrainz-server/blob/0ec0ddc/lib/MusicBrainz/Server/Controller/ReleaseEditor.pm#L606 to `$id + 0` so Perl treats it as a number.  If you can revert this & test that I'd prefer making the change on the server, because `linkTypeID` should never be a string according to its Flow type -- that's a server bug.

Reply:
It is in the release editor.

Reply:
Looks like in that case it's being parsed from the URL query string, so it's expected to be a string.